dotCMS/src/main/java/com/dotcms/publisher/util/PublisherUtil.java

```diff
@@ -61,8 +61,12 @@ public static PublishingEndPoint getObjectByMap(Map<String, Object> row){
 			pep.setGroupId(row.get("group_id").toString());
 		}
 		pep.setAddress(row.get("address").toString());
-		pep.setPort(row.get("port").toString());
-		pep.setProtocol(row.get("protocol").toString());
+		if(row.get("port") != null) {
+		    pep.setPort(row.get("port").toString());
```

Comment:
Another option was to do something like this: pep.setPort(String.valueOf(row.get("port”)); 
Suggested by @wezell